core/components/atoms/collapsible/Collapsible.tsx

```diff
@@ -56,6 +56,7 @@ export const Collapsible = (props: CollapsibleProps) => {
 
   const BodyClass = classNames({
     ['Collapsible-body']: true,
+    ['Collapsible-body--hoverable']: !expanded && hoverable,
```

Comment:
Can we replace the class name with our design token **overflow-hidden**? Please check.